### PR TITLE
[Az.ApplicationInsights] Fix DocumentType for New-AzApplicationInsightsContinuousExport

### PR DESCRIPTION
**Pull request by VeryEarly**
## Description

#18350 
#18707 

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Review by divya-jay — COMMENTED** [review]
I believe the same fix is required on the Set-AzApplicationInsightscontinuousExport cmdlet

**Comment by VeryEarly** [comment]
> I believe the same fix is required on the Set-AzApplicationInsightscontinuousExport cmdlet

nice catch

File: src/ApplicationInsights/custom/Set-AzApplicationInsightsContinuousExport.ps1 [hunk 1]
```diff
@@ -75,9 +75,10 @@ function Set-AzApplicationInsightsContinuousExport {
         [Parameter()]
         [Microsoft.Azure.PowerShell.Cmdlets.ApplicationInsights.Category('Body')]
         [System.String[]]
+        [Alias('DocumentType ')]
```
Comment (dingmeng-xue) [inline src/ApplicationInsights/custom/Set-AzApplicationInsightsContinuousExport.ps1:78]: Remove it.

File: src/ApplicationInsights/docs/New-AzApplicationInsightsContinuousExport.md [hunk 2]
```diff
@@ -70,13 +70,14 @@ Accept pipeline input: False
...
-
+### -DocumentType
+The document types to be exported, as comma separated values.
+Allowed values include 'Requests', 'Custom Event', 'Exception', 'Metric', 'Page View', 'Page Load', 'Dependency', 'Performance Counter', 'Availability', 'Trace'.
 
 ```yaml
 Type: System.String[]
 Parameter Sets: (All)
-Aliases:
+Aliases: DocumentType
```
Comment (dingmeng-xue) [inline src/ApplicationInsights/docs/New-AzApplicationInsightsContinuousExport.md:80]: should remove

File: src/ApplicationInsights/custom/New-AzApplicationInsightsContinuousExport.ps1 [hunk 3]
```diff
@@ -79,9 +79,10 @@ function New-AzApplicationInsightsContinuousExport {
         [Parameter()]
         [Microsoft.Azure.PowerShell.Cmdlets.ApplicationInsights.Category('Body')]
         [System.String[]]
+        [Alias('DocumentType ')]
         # The document types to be exported, as comma separated values.
-        # Allowed values include 'Requests', 'Event', 'Exceptions', 'Metrics', 'PageViews', 'PageViewPerformance', 'Rdd', 'PerformanceCounters', 'Availability', 'Messages'.
-        ${DocumentType },
+        # Allowed values include 'Requests', 'Custom Event', 'Exception', 'Metric', 'Page View', 'Page Load', 'Dependency', 'Performance Counter', 'Availability', 'Trace'.
```
Comment (dingmeng-xue) [inline src/ApplicationInsights/custom/New-AzApplicationInsightsContinuousExport.ps1:84]: Please correct it

Comment (VeryEarly) [reply src/ApplicationInsights/custom/New-AzApplicationInsightsContinuousExport.ps1:83]: description for this parameter in Set-AzApplicationInsightsContonuousExport was corrected

**Review by dingmeng-xue — APPROVED** [review]

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**